umbra-js/src/utils/utils.ts

```diff
@@ -586,3 +586,5 @@ export async function assertSupportedAddress(recipientId: string) {
   if (await contract.isSanctioned(address)) throw new Error(errMsg);
   return true;
 }
+
+export const jsonFetch = (url: string) => fetch(url).then((res) => res.json());
```

Comment:
Can we keep this `jsonFetch` definitionin the frontend? Because:
1. That's where we use it, and it's not really related to this umbra-js file
2. It uses fetch, which is defined in browsers (i.e. frontend folder) but doesn't exist in node environments. The umbra-js code should work in both browsers and node, and this breaks that

Reply:
Ah yes, removed from umbra-js